python/orca/tutorial/xshards/notebooks/diabetes.ipynb

```diff
@@ -376,9 +382,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "py37tf2_x",
+   "display_name": "yinchen",
```

Comment:
revert this?

Reply:
oh no, vscode automatically change this 😢